utils/uuid.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+)
+
+// UUID will generate a random v14 unique identifier based upon random numbers
+// nolint:gomnd
+func UUID() string {
```

Comment:
Why was `github.com/google/uuid` not used for uuid generation by the way?

Reply:
I think it was primarily to avoid an additional dependency.

Reply:
This implementation could be simply replaced by `uuid.New().String()`, that might worth a new PR

Reply:
@mmorel-35 A follow up PR after this to change that is welcome.